src/transformers/models/fuyu/modeling_fuyu.py

```diff
@@ -174,9 +174,11 @@ def get_placeholder_mask(
     @auto_docstring
     def forward(
         self,
-        input_ids: torch.LongTensor = None,
-        image_patches: torch.Tensor = None,  # [batch_size, num_total_patches, patch_size_ x patch_size x num_channels ]
-        image_patches_indices: torch.Tensor = None,
+        input_ids: Optional[torch.LongTensor] = None,
+        image_patches: Optional[
+            torch.Tensor
+        ] = None,  # [batch_size, num_total_patches, patch_size_ x patch_size x num_channels ]
```

Comment:
this layout is distressing - maybe just remove the comment or move it to its own line somewhere?